scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -118,7 +118,7 @@
       </table>
     </div>
     <div class="card-footer">
-      <strong>Cases</strong>
+      <a href="#" data-toggle="tooltip" title="Show maximum 10 cases"><strong>Cases</strong></a>
```

Comment:
👍

Reply:
I'll merge this soon

Reply:
No, not with that forcera-pushen Berge.

Reply:
😂

Reply:
Sorry, check my merge on the changelog. I fixed it for you once, but you forced over it. You are rewriting history... 😉

Reply:
🤣 it never ends